conf/machine/jetson-agx-orin-devkit.conf

```diff
@@ -33,3 +33,5 @@ tegra234-p3737-camera-e3331-overlay.dtbo,\
 tegra234-p3737-camera-e3333-overlay.dtbo,\
 tegra234-p3737-camera-imx185-overlay.dtbo,\
 tegra234-p3737-camera-imx390-overlay.dtbo"
+
+MACHINE_EXTRA_RRECOMMENDS += "kernel-module-rtk-btusb kernel-module-rtl8822ce tegra-firmware-rtl8822"
```

Comment:
Hey @kekiefer 
It is not preferable to update the `MACHINE_EXTRA_RRECOMMENDS` in `conf/machine/include/tegra234.inc` ?
Please share your thoughts.

Reply:
Because this is something specific to the devkit, I chose to put it here. Custom machines would include `tegra234.inc` and would not necessarily have this wifi module.

Reply:
Agree, you are right :slightly_smiling_face: